community/values/src/main/java/org/neo4j/values/storable/Values.java

```diff
@@ -253,6 +257,40 @@ public static ShortArray shortArray( short[] value )
         return new ShortArray.Direct( value );
     }
 
+    public static PointValue pointValue( CoordinateReferenceSystem crs, double... coordinate )
+    {
+        return new PointValue( crs, coordinate );
+    }
+
+    public static PointValue point( Point point )
+    {
+        // An optimization could be to do an instanceof PointValue check here
+        // and in that case just return the casted argument.
+        List<Double> coordinate = point.getCoordinate().getCoordinate();
+        double[] coords = new double[coordinate.size()];
+        for ( int i = 0; i < coords.length; i++ )
+        {
+            coords[i] = coordinate.get( i );
+        }
+        return new PointValue( crs( point.getCRS() ), coords );
+    }
+
+    public static CoordinateReferenceSystem crs( CRS crs )
+    {
+        if ( crs.getHref().equals( CoordinateReferenceSystem.WGS84.href ) )
```

Comment:
would prefer to have condition flipped to have constants on the left side to avoid NPEs in any case independently from crs that can be passed here ever